src/System.IO.FileSystem/src/MatchingRefApiCompatBaseline.txt

```diff
@@ -1,7 +1,8 @@
 Compat issues with assembly System.IO.FileSystem:
+# These are now virtual in the implementation, which should be ok
 CannotMakeMemberAbstract : Member 'System.IO.FileSystemInfo.Exists' is abstract in the implementation but is not abstract in the contract.
 CannotMakeMemberAbstract : Member 'System.IO.FileSystemInfo.Name' is abstract in the implementation but is not abstract in the contract.
 CannotMakeMemberAbstract : Member 'System.IO.FileSystemInfo.Exists.get()' is abstract in the implementation but is not abstract in the contract.
 CannotMakeMemberAbstract : Member 'System.IO.FileSystemInfo.Name.get()' is abstract in the implementation but is not abstract in the contract.
-TypesMustExist : Type 'System.IO.Enumeration.FileSystemEntry.<_fileNameBuffer>e__FixedBuffer' does not exist in the implementation but it does exist in the contract.
-MembersMustExist : Member 'System.IO.Enumeration.FileSystemEnumerator<TResult>.GetData()' does not exist in the implementation but it does exist in the contract.
+# Not sure why this one is coming up, it is being generated by C# for the Unix version
+TypesMustExist : Type 'System.IO.Enumeration.FileSystemEntry.<_fileNameBuffer>e__FixedBuffer' does not exist in the implementation but it does exist in the contract.
```

Comment:
This looks like a bug in the C# compiler. This type should not be public. 
cc @VSadov

Reply:
Probably. The backing types for fixed buffers are emitted as public, but if the field is not public, then the type does not need to be. I'd guess the same accessibility as on the field would make sense.

This could be a very old bug though. I wonder if it would be safe to fix now.